recipes/rust-overlaps/build.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash -e
 
+# circumvent a bug in conda-build >=2.1.18,<3.0.10
```

Comment:
We should be able to remove these workarounds by now, actually. `conda-build >=3.0.10` shouldn't be adding those empty-value environment variables anymore.